draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3456,808 +3548,1280 @@ ObjectLists MAY combine regular objects and secondary ObjectLists in the same ob
+Accept: */*
 
 RESPONSE:
 
-  HTTP/1.1 200 OK
-  Content-Length: 152
-  Expires: Wed, 04 May 2016 08:49:11 GMT
-  Server: example-server/0.1
-  ETag: "4331492443626270781"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:11 GMT
-  Content-Type: application/cdni; ptype=ci-trigger-collection
+HTTP/1.1 200 OK
+Content-Length: 191
+Expires: Sun, 27 Oct 2024 08:49:19 GMT
+Server: example-server/0.1
+ETag: "4331492443626270781"
+Last-Modified: Sun, 27 Oct 2024 08:40:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:19 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
 
-  {
-    "staleresourcetime": 86400,
-    "triggers": [
-      "https://dcdn.example.com/triggers/0",
-      "https://dcdn.example.com/triggers/1"
-    ]
-  }
+{
+  "staleresourcetime": 86400,
+  "triggers": [
+    "https://dcdn.example/cit/3f2d259d-a980-4742-beeb-9392a58129f5",
+    "https://dcdn.example/cit/327df5b8-1df8-4cff-92f8-fda27774c171"
+  ]
+}
 ]]></artwork>
    </figure>
    <t>
    At this point, if no other triggers had been created,
-   the other filtered views would be empty. For example:</t>
+   the other filtered representations would be empty. For example:</t>
 
    <figure><artwork><![CDATA[
 REQUEST:
 
-  GET /triggers/complete HTTP/1.1
-  User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
-  Accept: */*
+GET /cit/state/complete HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
 
 RESPONSE:
 
-  HTTP/1.1 200 OK
-  Content-Length: 54
-  Expires: Wed, 04 May 2016 08:49:11 GMT
-  Server: example-server/0.1
-  ETag: "7958041393922269003"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:11 GMT
-  Content-Type: application/cdni; ptype=ci-trigger-collection
+HTTP/1.1 200 OK
+Content-Length: 51
+Expires: Sun, 27 Oct 2024 08:49:20 GMT
+Server: example-server/0.1
+ETag: "7958041393922269003"
+Last-Modified: Sun, 27 Oct 2024 08:48:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:20 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
 
-  {
-    "staleresourcetime": 86400,
-    "triggers": []
-  }
+{
+  "staleresourcetime": 86400,
+  "triggers": []
+}
 ]]></artwork>
-   </figure>
-   </section>
 
-   <section title="Individual Trigger Resources" anchor="exmaples-trigger-resources"><t>
-   uCDN can also examine individual triggers. For example, for the "preposition" and "invalidate" 
-   triggers from previous examples:</t>
+     </figure>
+     </section>
+
+     <section title="Individual Trigger Resources" anchor="examples-trigger-resources">
+     <t>
+     uCDN can also examine individual triggers. For example, for the "preposition" and "invalidate" 
+     triggers from previous examples:
+     </t>
 
 
-   <figure><artwork><![CDATA[
+     <figure><artwork><![CDATA[
 REQUEST:
 
-  GET /triggers/0 HTTP/1.1
-  User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
-  Accept: */*
+GET /cit/3f2d259d-a980-4742-beeb-9392a58129f5 HTTP/1.1
```

Comment:
Ignoring the fact that earlier, we canceled and then deleted this one... :-)

Reply:
fixed :)